<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCHEConfig</class>
 <widget class="QDialog" name="PCHEConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>640</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>95</y>
     <width>431</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/PCHEImg1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>432</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/PCHEImg2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>88</y>
     <width>83</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>P Outlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pAirOut"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>161</y>
     <width>75</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>T Inlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tAirIn"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>161</y>
     <width>83</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>T Outlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tAirOut"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>55</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Number of ZigZags</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="nbZigZag"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>265</y>
     <width>81</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="lBox">
      <property name="suffix">
       <string> mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>55</y>
     <width>75</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Diff. P Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="diffP"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>T Inlet Channel 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tWIn1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>P Inlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pWIn"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>T Inlet Channel 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tWIn2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>T Outlet Channel 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tWOut1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>T Outlet Channel 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tWOut2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>500</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>P Outlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pWOut"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Channel Height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="chHeight">
      <property name="suffix">
       <string> mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_14">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Channel Width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="chWidth">
      <property name="suffix">
       <string> mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_15">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_17">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Channel number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="nbChannels"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>82</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-- Air --</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>118</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-- Water --</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_16">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_18">
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Flow Rate Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="flowRate"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_17">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>91</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_19">
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>T ambient Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tAmb"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_18">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_20">
    <item>
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Angle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="angle">
      <property name="suffix">
       <string> °</string>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_19">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_21">
    <item>
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Wall thickness</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="wThick">
      <property name="suffix">
       <string> mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>75</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>P Inlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pAirIn"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PCHEConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PCHEConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
